packages/salesforcedx-vscode-apex-replay-debugger/package.json

```diff
@@ -191,7 +218,7 @@
       {
         "type": "apex-replay",
         "label": "Apex Replay Debugger",
-        "program": "./node_modules/@salesforce/salesforcedx-apex-replay-debugger/out/src/adapter/apexReplayDebug.js",
+        "program": "./dist/apexreplaydebug.js",
```

Comment:
oh! this is probably why integration tests were requiring a bundle. I'll see what I can do about it.

Reply:
Can you say more about why you think this is why the integration tests required bundling?

Reply:
yeah so basically this tells the extension where to find the js file to launch as the debugger server.  If that file doesn't exist the extension will fall over and say the debugging server crashed.  I was seeing that in circle and confused by it b/c it was unable to find the 'dist/*.js' file.   If the bundling scripts are run then no `dist/apexreplaydebug.js` files exists b/c that script is what bundles and copies over the result the dist folder. 

The int test then fails b/c one of the extensions isn't running and we have some int tests that use command calls to execute flows across extensions. 

I started looking at this locally and found I already accounted for fixing the issue, but forgot to back out the hardcoded change.  commit coming shortly.